packages/lib-classifier/src/store/DrawingStore.js

```diff
@@ -76,49 +50,21 @@ const DrawingStore = types
       addDisposer(self, workflowStepsDisposer)
     }
 
-    function createMark () {
-      const MarkModel = getMarkModel(self.activeDrawingTask.tools[self.activeDrawingToolIndex].type)
+    function createMark (event) {
+      const activeTool = self.activeDrawingTask.tools[self.activeDrawingToolIndex]
 
       const tempId = cuid()
 
-      const newMark = MarkModel.create({
+      const newMark = activeTool.createMark({
         id: tempId,
```

Comment:
I don't think the unique id is necessary anymore if a reference to it is not being used in the stores.

Reply:
I think `marks` is still a map, which needs an identifier? This will probably come in useful once we're selecting and editing marks.

Reply:
Oh I see, yes.

Reply:
Then, we'll need a reference for selecting one out of the map? Removing the `safeReference` type may be premature?

Reply:
It should still be there.

Reply:
Oh wait. I took the active mark out of the store. It's in component state now.

Reply:
This ID is used as a key when rendering marks, so I'm going to leave it in.